iabtcf-decoder/src/test/java/com/iabtcf/decoder/TCStringV1Test.java

```diff
@@ -75,15 +70,15 @@ public void testAllowedPurposes() {
         TCString model = parse("BOOzQoAOOzQoAAPAFSENCW-AIBA=");
         assertThat(model.getPurposesConsent(), matchInts(1, 2, 3, 4, 5, 15, 24));
         assertTrue(model.getPurposesConsent().contains(1));
-        assertTrue(model.getPurposesConsent().contains(STORAGE_AND_ACCESS.getId()));
+        assertTrue(model.getPurposesConsent().contains(1));
+        assertTrue(model.getPurposesConsent().contains(2));
         assertTrue(model.getPurposesConsent().contains(2));
-        assertTrue(model.getPurposesConsent().contains(PERSONALIZATION.getId()));
         assertTrue(model.getPurposesConsent().contains(3));
-        assertTrue(model.getPurposesConsent().contains(AD_SELECTION.getId()));
+        assertTrue(model.getPurposesConsent().contains(3));
         assertTrue(model.getPurposesConsent().contains(4));
-        assertTrue(model.getPurposesConsent().contains(CONTENT_DELIVERY.getId()));
+        assertTrue(model.getPurposesConsent().contains(4));
+        assertTrue(model.getPurposesConsent().contains(5));
```

Comment:
I see duplicate assertions. any reason for that?

Reply:
good catch, I'll address this.